QA_Requests/PerfTickets/testCases/test_generate_upload_subquery_data.py

```diff
@@ -0,0 +1,17 @@
+from QA_Requests.PerfTickets.testPages.subcase_query.generate_data import generate_path, data_generation, \
+    subcase_data_generation
+from QA_Requests.PerfTickets.userInputs.test_data import TestData
+
+
+def test_case_dummy_data(driver):
+    col_list = ['number', 'external_id', 'name', 'show_name']
+    child_col_list = ['number', 'caseid', 'name', 'location', 'multi_data_1', 'receive_geopoint_1',
+                      'parent_external_id', 'parent_type',
+                      'hundred', 'two_hundred', 'three_hundred', 'four_hundred', 'five_hundred']
+    global load, subcase
+    load = 1000000
```

Comment:
again the same question, why are we declaring global vars for only 1 function?

Reply:
I made some changes and didn't remove those later -- Removed now https://github.com/dimagi/dimagi-qa/pull/137/commits/8fa03009d5d9714d8a4724bb34c4ee738d0710dc